<launch> 
  <include file="$(find robot_localization)/launch/t265_handler.launch"/>
  <include file="$(find mav_regulator)/launch/px4.launch"/>

  <node name="mav_regulator_node" pkg="mav_regulator" type="mav_regulator_node" output="screen">
    <param name="mav/gravity"             value="9.8055"/>
    <param name="mav/T_max"               value="25.0"/>
    <param name="mav/T_min"               value="3.0"/>
    <param name="mav/w_xy_max"            value="1.0"/>
    <param name="mav/w_z_max"             value="1.0"/>

    <param name="mpc/q_px"                value="200.0"/>
    <param name="mpc/q_py"                value="200.0"/>
    <param name="mpc/q_pz"                value="200.0"/>

    <param name="mpc/q_qw"                value="80.0"/>
    <param name="mpc/q_qx"                value="80.0"/>
    <param name="mpc/q_qy"                value="80.0"/>
    <param name="mpc/q_qz"                value="80.0"/>

    <param name="mpc/q_vx"                value="40.0"/>
    <param name="mpc/q_vy"                value="40.0"/>
    <param name="mpc/q_vz"                value="40.0"/>

    <param name="mpc/q_th"                value="0.2"/>
    <param name="mpc/q_wx"                value="1.6"/>
    <param name="mpc/q_wy"                value="1.6"/>
    <param name="mpc/q_wz"                value="1.6"/>

    <param name="mpc/dt"                  value="0.05"/>
  </node>

  <node name="performance_eval_node" pkg="mav_regulator" type="performance_eval_node" output="screen">
    <param name="execution_time"    value="4.0"/>
    <param name="takeoff_altitude"  value="1.0"/>
    <param name="goal_thr"          value="0.1"/>
    
    <param name="trajectory"        value="$(find mav_regulator)/eval_trajs/trajectory_circle.yaml"/>
  </node> 
</launch>
